assets/script.js

```diff
@@ -110,6 +110,10 @@ function getAlbumInfo() {
         //album search result shown
         $("#album-pic").attr("src", icon);
         $("#summaryHeading").text(albumName);
+        $("#album-pic").attr("title", "Artist: " + response.album.artist);
+        $("a.img").each(function () {
```

Comment:
This selects all a tags with .img class so it's not doing anything

Reply:
@Goldenaries658 I need to keep this in because its targeting the img class i added to all the album modal-triggers

Reply:
No worries, I missed that 👍🏾